### ISSUE TITLE
Task Execution Failure for `:app:mergeDebugJavaResource` with `android-omh-auth`

### ISSUE THREAD

**Issue by dzuluaga** [openmobilehub/android-omh-auth#98]
When using the `android-omh-auth` library, the following error is generated during the build process:

```
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':app:mergeDebugJavaResource'.
...
Caused by: com.android.builder.merge.DuplicateRelativeFileException: 2 files found with path 'META-INF/DEPENDENCIES'.
Adding a packaging block may help, please refer to
https://developer.android.com/reference/tools/gradle-api/8.5/com/android/build/api/dsl/Packaging
for more information
```

#### Workaround
We have identified a workaround that resolves the issue by modifying the `build.gradle.kts` file at the app level:

```kotlin
packaging {
    resources.excludes.add("META-INF/DEPENDENCIES")
}
```

While this workaround is effective, ideally, users should not have to manually add this configuration to their projects.

#### Request
Please investigate a potential fix for this issue so that users do not need to manually add the packaging block. In the meantime, it would be helpful to document this workaround in the `android-omh-auth` documentation, similar to how it has been documented for `react-native-omh-auth` [here](https://openmobilehub.github.io/react-native-omh-auth/docs/getting-started/#2-files-found-with-path-meta-infdependencies).

**Comment by adamTrz** [comment]
Hi @dzuluaga 
I've extended docs with how users can fix this error - #106 

Unfortunately it looks like this is an official fix that is mentioned in multiple places like [here](https://stackoverflow.com/questions/44342455/more-than-one-file-was-found-with-os-independent-path-meta-inf-license/47509465#47509465) and [here (Auth0 GH)](https://github.com/auth0/Auth0.Android/issues/598)

**Comment by dzuluaga** [comment]
Thanks, @adamTrz. Please merge the changes to close this issue.
